src/App.js

```diff
@@ -1,25 +1,81 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+let DATA = [
+  {
+    id: 1,
+    title: "Lorem ipsum dolor sit amet consectetur",
+  },
+  {
+    id: 2,
+    title: "Lorem ipsum dolor sit amet consectetur",
+  },
+];
```

Comment:
It would be better if you stick with the actual user data instead of starting with lorem data